tools/ci_build/github/azure-pipelines/templates/jobs/win-ci-prebuild-steps.yml

```diff
@@ -0,0 +1,94 @@
+parameters:
+- name: EnvSetupScript
+  type: string
+
+- name: BuildConfig
+  type: string
+
+- name: BuildArch
+  type: string
+
+- name: DownloadCUDA
+  type: boolean
+  default: false
+
+- name: WITHCACHE
+  type: boolean
+  default: false
+
+- name: MachinePool
+  type: string
+
+- name: Today
+  type: string
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.8'
+    addToPath: true
+    architecture: ${{parameters.BuildArch}}
+
+- script: |
+    python -m pip install -q setuptools wheel numpy
```

Comment:
In python packaging pipeline, we need to fix the numpy version. The other pipelines do not need to.